morevna-ep4.msrt

```diff
@@ -201,12 +201,12 @@
 41
 00:05:23,000 --> 00:05:26,220
 [rus] *Эй, сюда! В сторону, сторону! Давайте его сюда! Осторожнее!*
-[eng] *Here! Step aside, step aside! Let's get him over here! Careful!*
+[eng] *Over here! Step aside, step aside! Let's get him over here! Careful!*
 
 42
 00:05:26,280 --> 00:05:27,670
 [rus] *Ну-ка, разошлись все!*
-[eng] *Get out of my way!*
+[eng] *Move it kid!*
```

Comment:
"Move it!" would work fine. I added kid just to make it a bit more dismissive and so it better matches the length of the Russian speech.

Reply:
The literal meaning here is "Okay, stay away everyone!" (so it is not addressed to "kid", but to "everyone".

Reply:
I see. So do you think "Move it!" would be appropriate or do you want to go for something else?

Reply:
Let's think about something else. ^__^

Reply:
Here is my suggestion: change this back to "Get out of my way!". And  use "Get out of here" below to avoid repetition.

Reply:
Yes that suggestions sounds good to me.